recipes/hyper/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "hyper" %}
+{% set version = "0.7.0" %}
+{% set sha256 = "069514f54231fb7b5df2fb910a114663a83306d5296f588fffcb0a9be19407fc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
You'll want to [add the entry point](https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points) too:
```
  entry_points:
    - hyper = hyper.cli:main
```

Reply:
(Note: done.)